src/Uno.Toolkit.UI/Behaviors/FlipViewExtensions.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.System;
+using Windows.UI.ViewManagement;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+#endif
+
+namespace Uno.Toolkit.UI;
+
+public static class FlipViewExtensions
+{
+	#region DependencyProperty: Next
+	public static DependencyProperty NextProperty { get; } =
+	DependencyProperty.RegisterAttached("Next", typeof(FlipView), typeof(FlipViewExtensions), new PropertyMetadata(null, OnNextChanged));
+
+	public static void SetNext(Button element, FlipView value) => element.SetValue(NextProperty, value);
+
+	public static FlipView GetNext(Button element) => (FlipView)element.GetValue(NextProperty);
```

Comment:
As [nullability](https://github.com/unoplatform/uno.toolkit.ui/blob/d716fe4abe02ed8f69aacb06ff440e6fb90ea4cd/src/Directory.Build.props#L20) is enabled:
```suggestion
	public static FlipView? GetNext(Button element) => (FlipView?)element.GetValue(NextProperty);
```
Same for the other properties:

@kazo0 did you handle this in the other behaviors that were created recently?